server/controllers/profileController.js

```diff
@@ -0,0 +1,70 @@
+const Profile = require("../models/ProfileModel");
+const mongoose = require("mongoose");
+
+const toObjectId = mongoose.Types.ObjectId;
+
+// create new profile using given arguments
+
+exports.createProfile = async (req, res, next) => {
+  try {
```

Comment:
Please validate the logged in user here: `req.user.id` (Make sure you refer to the comment below)

Eg:
```
const userId = req.user.id;
const user = User.findById(userId);
if (!user) {
res.send(401);
throw new Error("Not authorized");
}
```